mathics/core/expression.py

```diff
@@ -270,6 +270,12 @@ def _build_elements_properties(self):
         values = []
         for element in self._elements:
             # Test for the literalness, and the three properties mentioned above
+            try:
```

Comment:
Is this something that is going to stay here?

Reply:
Thanks for catching!  removed now.